### DISCUSSION TITLE
Platform Green Support

### DISCUSSION THREAD

**Discussion by JailbreakPapa** [jrouwe/JoltPhysics#734] [Q&A] [answered]
Hello, My team recently added _Platform Blue_ support to our source of Jolt, and we are wondering if the process would be the same for adding the Physics Engine over to _Platform Green_, And if there are any glaring changes that i would have to make for _Platform Green._

**Comment by mihe** [comment, marked as answer]
You might want to check out #711.

**Reply by jrouwe** [reply]
To add to that: For Platform Green I would expect it to be almost no work since it's very similar to PC. If you get it to work (and make a ```JPH_PLATFORM_GREEN``` define) then I'd be interested to integrate this into the main repository (make sure you don't breach any NDA's by including platform specific APIs though).

**Reply by JailbreakPapa** [reply]
> To add to that: For Platform Green I would expect it to be almost no work since it's very similar to PC. If you get it to work (and make a ```JPH_PLATFORM_GREEN``` define) then I'd be interested to integrate this into the main repository (make sure you don't breach any NDA's by including platform specific APIs though).

Yeah, I'll make a PR once we are done with porting it over to Xbox. Shouldn't be too difficult as you said.
